### ISSUE TITLE
libtorrent package required

### ISSUE THREAD

**Issue by once375ml** [Flexget/Flexget#3874]
Jackett offers a lot of Rss(es) for Flexget, but some torrent links are not reachable (banned by China GFW). However, I found the magnet works. Unfortunately, Flexget doesn't accept magnet directly. So I want to convert the magnet(s) into torrent(s). Then I got the below feedback from Flexget

"pip install libtorrent" does not work:
ERROR: Could not find a version that satisfies the requirement libtorrent (from versions: none)
ERROR: No matching distribution found for libtorrent

### Expected behaviour:

Plugin convert_magnet converts the magnet it meets

### Actual behaviour:

Plugin convert_magnet says：libtorrent package required

### Steps to reproduce:
- Step 1: ...

#### Config:
```yaml
Paste FULL config and remove any personal info if config is too long, attach the file to the ticket.
If issue is with a single task, you can get get resulting configuration by running:
  flexget execute --task <NAME> --dump-config
Make sure to redact any personal information (passwords, api keys, etc) !
```
  
#### Log:
<details>
 <summary>(click to expand)</summary>

```
paste log output here
```
</details>

### Additional information:

- FlexGet version:
- Python version:
- Installation method:
- Using daemon (yes/no): yes
- OS and version: Windows 10
- Link to crash log:

**Comment by gazpachoking** [comment]
What do you mean flexget doesn't support the magnet directly? All of our torrent client plugins support them as far as I'm aware. libtorrent is not a pure python component, so it is harder to install.

**Comment by once375ml** [comment]
> What do you mean flexget doesn't support the magnet directly? All of our torrent client plugins support them as far as I'm aware. libtorrent is not a pure python component, so it is harder to install.

I would like Flexget to download from a Jackett gathered RSS, in which torrent direct url has been banned but magnet works.
The problem is Flexget can't pass the magnet links to downloader

**Comment by once375ml** [comment]
> What downloader are you using?

Download Station of Synology. Recently, I give up the test. So plz ignore this post.

**Comment by elfez** [comment]
I solved this by allowing the flexget venv to access system libraries. I did by editing `~/.local/pipx/venvs/flexget/pyvenv.cfg` and changing the line `include-system-site-packages` to read `true`.

It can then access the `python3-libtorrent` package I've installed on the debian system.

I do have a vague recollection of fixing this on a previous flexget installation by running pip from within the venv, but couldn't get it to work this time. 

Also, enabling access to system packages my or may not do horrible things to flexget. But it works for me.
